frontend/app/src/pages/main/v1/workflow-runs-v1/hooks/workflow-details.tsx

```diff
@@ -17,6 +29,15 @@ export const useWorkflowDetails = () => {
 
       return true;
     },
+    refetchInterval: (query) => {
+      const data = query.state.data;
+
+      if (isTerminalState(data?.run?.status)) {
```

Comment:
is this the only place? what about the task page?

Reply:
yeah can change that hook too, one sec

Reply:
well actually, so that one fetches every 5 seconds - do we want it to fetch more often than that? it'd be a ton of API calls

Reply:
no i meant for the detail pages --we have task-run and workflow-run pages, right?

Reply:
ah yeah good call, just fixed that - I'll factor that query out into a hook later, it's used a lot of places